Tests/Functional/Sources/ErrorHandlingTest.m

```diff
@@ -236,6 +236,15 @@ - (void)testKeyboardDismissalError {
   XCTAssertTrue([error.description containsString:keyboardErrorString]);
 }
 
+/// kGREYInteractionAssertionFailedErrorCode
```

Comment:
We try to comment new / modified tests as we proceed.

/**
 * Verifies the printed information for a GREYError found for an assertion failure.
 */